src/lib/utils/template_generator.js

```diff
@@ -128,6 +128,18 @@ class TemplateGenerator {
   getExternalProject(uri) {
     let url, folder, hgi;
     let fallback, url_fallback, folder_fallback, hgi_fallback, embarkVersion;
+    const replacements = {
+      'git@github.com/': 'git@github.com:',
+      'git@bitbucket.org/': 'git@bitbucket.org:',
+      'git@gitlab.com/': 'git@gitlab.com:',
+      'github.com/': '',
```

Comment:
what happens in that case?
We replace with nothing?

Reply:
In that case, the `hosted-git-info` library picks it up as a github shortcut and is then able to parse it properly.